recognize_people_srv_name: "detect_people"
probability_threshold: 0.2
link_threshold: 0.5
heuristic: "shoulder"
arm_norm_threshold: 0.5
neck_norm_threshold: 0.7
waving_threshold: 0.2
vert_threshold: 0.7
hor_threshold: 0.4
padding: 5
